dev/util.h

```diff
@@ -1,10 +1,53 @@
 #pragma once
 
 #include <sqlite3.h>
-#include <string>  //  std::string
+#include <string>  //  std::basic_string
+#include <utility>  //  std::move
+
+#include "error_code.h"
 
 namespace sqlite_orm {
 
+    /** 
+         *  Escape the provided character in the given string by doubling it.
+         *  @param str A copy of the original string
+         *  @param char2Escape The character to escape
+         */
+    inline std::string sql_escape(std::string str, char char2Escape) {
+        for(size_t pos = 0; (pos = str.find(char2Escape, pos)) != std::string::npos; pos += 2) {
```

Comment:
btw `std::string::npos` can be replaced with `str.npos`

Reply:
Yes it can, but it doesn't have to :)